README.md

```diff
@@ -38,9 +38,15 @@ new WC_Subscriptions_Core_Plugin();
 
 This repository can also be loaded as standalone plugin, for development purposes only.
 
+### Prerequisites
```

Comment:
This wording is inspired by the [WC README Prerequisites](https://github.com/woocommerce/woocommerce/blob/trunk/README.md#prerequisites)